galley/pkg/config/analysis/msg/messages.yaml

```diff
@@ -37,6 +37,26 @@ messages:
       - name: detail
         type: string
 
+  - name: "MisplacedAnnotation"
+    code: IST0005
```

Comment:
Is there a reason you're putting these in the reserved "internal" range? (See the comment at the top of `messages.yaml`). 

If not, the codes should be `IST0107` / `IST0108` (the next ones in sequence outside the reserved range).

Reply:
Done